### PR TITLE
 🌟 Rebuild user interface and Improve user experience

### LINKED ISSUE #40: Improve and rebuild user experience
Revamps the interface to make it more intuitive and clean, which allows user to find the necessary information easily.

### LINKED ISSUE #41: Make user interface responsive
Responsive design allows the application to looks good on devices with any screen size (larger than certain minimum threshold)

### PR DESCRIPTION
**Pull request by Ang-YC**
Revamps the interface to make it more intuitive and clean, which allows user to find the necessary information easily.

In this week, v1.2 milestone
- Logo are created to do product branding.
- All FXML and CSS are completely rewritten to adapt with the mockup to improve the cleanliness of the interface, and also to allow responsiveness by using `AnchorPane` for relative positioning.
- Window event function such as resize, drag, and maximize are recreated to mimic the original function because borderless window doesn't support them out of the box.
- Scroll bar, split pane, dialog pane and progress are overridden with custom style to adapt with our theme.
- Window are now Borderless / Undecorated to adapt with custom (and nice) Title Bar.
- Info Panel now shows the personal information of applicants when selected instead of a WebView Browser panel.
- UI are now responsive as long as the minimum width and height are 800 and 600 respectively. For example, Info Panel will show only single panel when its width is smaller than 550px, else it will be a split panel.
- All planned fields are being added into the UI, where unimplemented field default to `-` at the moment until it is being implemented.
- Unit and Integration Tests for most of the new UI and functionality are being added.

This PR will fix #40 and fix #41 when merged

### CODE REVIEW

File: src/test/java/systemtests/AddressBookSystemTest.java [hunk 1]
```diff
@@ -39,8 +33,8 @@
 import seedu.address.model.AddressBook;
 import seedu.address.model.Model;
 import seedu.address.testutil.TypicalPersons;
-import seedu.address.ui.BrowserPanel;
 import seedu.address.ui.CommandBox;
+import seedu.address.ui.InfoPanel;
 
 /**
  * A system test class for AddressBook, which provides access to handles of GUI components and helper methods
```
Comment (kexiaowen) [inline src/test/java/systemtests/AddressBookSystemTest.java]: Can change AddressBook to HR+ :)

Comment (Ang-YC) [reply src/test/java/systemtests/AddressBookSystemTest.java]: We can have a separate PR to convert all AddressBook to HR+ 😄

**Comment by Ang-YC** [comment]
This PR fixed #40 and #41

File: src/test/java/systemtests/AddressBookSystemTest.java (hunk 1 above)
Comment (kexiaowen) [reply src/test/java/systemtests/AddressBookSystemTest.java]: 👌🏼👌🏼

**Pull request merged**